testing/unittests/jlibtests.cpp

```diff
@@ -4724,6 +4724,7 @@ class TopNStressTest : public CppUnit::TestFixture
         x tally;
         for (unsigned activityId = 1; activityId <= NumIters; activityId++)
         {
+            // coverity[suppress:overflow_const] - deliberate overflow use case
```

Comment:
Is this annotation has same effect as similar mentioned in https://documentation.blackduck.com/bundle/coverity-docs/page/customizing_coverity/topics/annotations/c_annotations_c_cpp_code-line.html ?
```
//coverity[overflow_const : SUPPRESS]
```
The space between the  leading '//' and 'coverity...' is acceptable? (There is no space in the examples and I can't find any information about it.)

Reply:
I'm pretty sure any comment format is permitted (it does give an example using /* coverity... */)

But should have been "coverity[overflow_const : SUPPRESS]".
Now changed, squashed in.